### PR TITLE
feat: add loader bb-15

### PR DESCRIPTION
**Pull request by yulyaolshanska**
### Summary
This PR implements the loader component for the application.

### Created
- Loader component
- file for color variables

### Changes
- **app.tsx:** Added loader

image

### CODE REVIEW

**Review by fshabanov — CHANGES_REQUESTED** [review]
This is not a loader, this is a circular progress bar. You needed to create a Loader, which will be used when we have some loading data. Example: https://loading.io/assets/mod/spinner/rolling/lg.gif

**Comment by yulyaolshanska** [comment]
> This is not a loader, this is a circular progress bar. You needed to create a Loader, which will be used when we have some loading data. Example: https://loading.io/assets/mod/spinner/rolling/lg.gif

but the percentage should remains as in the figma?

**Comment by fshabanov** [comment]
> > This is not a loader, this is a circular progress bar. You needed to create a Loader, which will be used when we have some loading data. Example: https://loading.io/assets/mod/spinner/rolling/lg.gif
> 
> but the percentage should remains as in the figma?

No, no percentage. The one in figma is not a loader, it is a circular progress bar, which will be used in a completely different part

**Comment by yulyaolshanska** [comment]
> > > This is not a loader, this is a circular progress bar. You needed to create a Loader, which will be used when we have some loading data. Example: https://loading.io/assets/mod/spinner/rolling/lg.gif
> > 
> > 
> > but the percentage should remains as in the figma?
> 
> No, no percentage. The one in figma is not a loader, it is a circular progress bar, which will be used in a completely different part

I thought this was a loader design 🤦‍♀️. Does that mean I can make a loader with any design?

**Comment by fshabanov** [comment]
> > > > This is not a loader, this is a circular progress bar. You needed to create a Loader, which will be used when we have some loading data. Example: https://loading.io/assets/mod/spinner/rolling/lg.gif
> > > 
> > > 
> > > but the percentage should remains as in the figma?
> > 
> > 
> > No, no percentage. The one in figma is not a loader, it is a circular progress bar, which will be used in a completely different part
> 
> I thought this was a loader design 🤦‍♀️. Does that mean I can make a loader with any design?

Yep, you can take something close to the app design

**Review by WilhelmWesser — APPROVED** [review]

**Review by revenets — APPROVED** [review]

**Comment by revenets** [comment]
Just wondering, can we use the same gradient color for the loader as we use for the buttons' hover state?

File: apps/frontend/src/libs/components/loader/loader.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import styles from "./styles.module.css";
+
+const Loader: React.FC = () => {
+	return (
+		<div className={styles["loaderContainer"]}>
+			<div className={styles["loader"]}></div>
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/loader/loader.tsx:8]:
```suggestion
			<div className={styles["loader"]} />
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Comment by yulyaolshanska** [comment]
> Just wondering, can we use the same gradient color for the loader as we use for the buttons' hover state?

image

Like this?

File: apps/frontend/src/libs/components/loader/loader.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/loader/loader.tsx:1]: We don't need this import in a web project

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Pull request merged**